### PR TITLE
Correct path to folder with themes

### PR DESCRIPTION
**Pull request by dukebarman**

### CODE REVIEW

**Comment by dukebarman** [comment]
Maybe this variable get from config?

**Comment by XVilka** [comment]
Fixed in https://github.com/radare/radare2/commit/5cd31d7aa44b8d35b9b6b869167c5884c5091280

File: libr/core/cmd_eval.c [hunk 1]
```diff
@@ -235,7 +235,7 @@ static int cmd_eval(void *data, const char *input) {
 					eprintf ("Something went wrong\n");
 				}
 			} else if (input[2]=='?') {
-				eprintf ("Usage: eco [themename]  ;load theme from /usr/share/radare2/0.10.2-git/cons/\n");
+				eprintf ("Usage: eco [themename]  ;load theme from /usr/share/radare2/0.10.4-git/cons/\n");
```
Comment (radare) [inline libr/core/cmd_eval.c]: Wtf srsly? There's last/ whichc is a symlinl... But cmon... This path is not hardcoded, you have vars to grab the prefix, datadir, destdir...

**Comment by radare** [comment]
This is a pretty awful fix. See comment

**Pull request closed without merging**